infrastructure/cdn-in-a-box/Makefile

```diff
@@ -130,6 +130,15 @@ $(TS_RPM): $(TS_DIST_RPM)
 $(ORT_RPM): $(ORT_DIST_RPM)
 	cp -f "$?" "$@"
 
+# RPM target aliases
+traffic_monitor/traffic_monitor.rpm: $(TM_RPM)
+traffic_ops/traffic_ops.rpm: $(TO_RPM)
+traffic_portal/traffic_portal.rpm: $(TP_RPM)
+traffic_router/traffic_router.rpm: $(TR_RPM)
+traffic_router/tomcat.rpm: $(TOMCAT_RPM)
+traffic_stats/traffic_stats.rpm: $(TS_RPM)
+cache/traffic_ops_ort.rpm: $(ORT_RPM)
```

Comment:
These are, unfortunately, lies. These recipes are only the truth if you're running `make` from within the CDN-in-a-Box directory. If you run it anywhere else, it doesn't produce the target, which is something we now support.

Reply:
So again, we could make these `.PHONY` if we're okay with `make` not being able to tell when it needs to rebuild them, or we could again restrict it to running within the CDN-in-a-Box directory if we're okay with the that restriction. Of course, it'd still need to support running in directories with spaces in them, since a user should be able to clone wherever they want and we don't have control over that. In that case, I'd be totally fine taking out the checks at the start of the makefile that it used to have to error out if you're not calling it from the right place, if those are too much of a pain to do correctly. I don't think people did that, in general, and I don't think `cd` as a dependency is asking too much.

Reply:
Gave the Makefile separate targets for relative paths and absolute paths in fc200a5946. Now it tab-completes both the relative-path RPM filename and the full-path RPM filename.

If you build a relative-path RPM target, Make treats the corresponding full-path RPM target as built, too.